packages/@lwc/template-compiler/src/__tests__/fixtures/expression-complex/throw/metadata.json

```diff
@@ -0,0 +1,13 @@
+{
+    "warnings": [
+        {
+            "code": 1052,
+            "message": "Unexpected compilation error: LWC1052: Error parsing attribute: Unexpected token (3:31)",
```

Comment:
This error message isn't really useful. The location `(3:31)` isn't valid as well.

Reply:
`(3:31)` refers to `throw` - that seems correct to me. As for the error message, I'm unsure how to improve it. `throw` is not valid in expressions; it is only parsed correctly as a statement. Because we've disallowed statements, I can't test that directly. And because the parser encountered a reserved keyword in an expression, "Unexpected token" is correct.

I'm unsure of the best way to resolve this. I suppose when "Unexpected token` is encountered, we could look at the indicated location to see if it is a `throw` string, but that feels somewhat hacky.